HelpSource/Guides/OSX_Standalones.schelp

```diff
@@ -0,0 +1,286 @@
+title:: OSX_Standalones
+summary:: Making simple standalones by copy-adapting existing apps.
+categories:: Utilities
+
+Note::
+Known working versions for making standalones:
+strong::SC3.8.0, SC3.9dev from branch [2a105f1]::
+code::
+"https://supercollider.github.io/download".openOS;
+::
+Self-built SC versions may have faulty userAppSupportDir behavior: sometimes code:: Platform.userAppSupportDir :: is code::.../SuperCollider/ :: and not the name of the newly made app.
+::
+written by Alberto de Campo, 2017
+
+section:: Introduction
+
+Guides/OSX_Standalones
+
+What makes Standalone applications useful differs depending on their intended use. They can range from slightly diverging copies of a full SuperCollider.app with IDE and help system to an app tweaked specifically for a certain user group, with a custom interface, and little of standard SC3 still visible.
+
+
+This Guide is for exporting copies of an existing SC3 setup,
+for at least two scenarios:
+
+A - Freezing an SC-based setup for non-expert users,
+say an installation for a piece that runs in a museum,
+which is as easily movable and selfcontained as possible.
+
+B - Copying a setup for a piece or performance setup,as backup
+or safety copy, keeping much flexibility for skilled users.
+
+Creating more complex user-oriented applications with SC was possible in versions pre 3.6, and a number of people work on additions to SC that will allow similar flexibility again.
+
+
+subsection:: How this guide works
+
+This set of scripts creates a SuperCollider-based
+Standalone app in a few basic steps:
+
+1. copy the open SuperCollider.app or SC3-derived standalone.
+
+2. fix the copy's info files so it uses its own userAppSupportDir
+
+3. test that Platform.userAppSupportDir is independent in the new app!
+
+Then the adaptation to the specific project begins:
+
+4. transfer active quarks and extensions to the new app
+
+5. copy the project folder used to the new app
+
+6. make an internal startupFile for it
+
+subsection:: What to prepare
+
+* have your system ready with all desired quarks installed,
```

Comment:
would be great to have such a minimal startup file listing here... something like 

```
s.waitForBoot{
	{
		Line.kr(1, 0, 10, doneAction: 2) * SinOsc.ar(1000) * EnvGen.ar(Env.perc(0.01, 0.25), Impulse.kr(2), 0.1)!2
		
	}.play;
}
```